app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -346,7 +348,9 @@ const SendBitcoinDetailsScreen: React.FC<Props> = ({ route }) => {
 
           const result = await fetchLnurlInvoice({
             lnUrlOrAddress: paymentDetail.destination,
-            tokens: btcAmount.amount as Satoshis,
+            // FIXME: remove any from lnurl-pay lib?
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            tokens: btcAmount.amount as any,
```

Comment:
Should we remove `Satoshis` from lnurl-pay library? 

This is what I have done for galoymoney-client. 

Those type are not very portable across library (or maybe I just don't know how to do it properly), so I think using number or bigint may be better. I've switch from Satoshis to number in `galoymoney-client`

Reply:
If you switched to number in `galoymoney-client` why do you need to cast `btcAmount.amount` as any? I think it could be beneficial to keep the Satoshi type and just export it from galoy-money client, or we could rename the field. Some of the libraries use millisatoshis, so it's important we know which one is required.

Reply:
well, not sure about the config in react but types should be loaded automatically https://github.com/dolcalmi/lnurl-pay/blob/main/package.json#L8 :thinking:

Reply:
the issue I faced is that Satoshi in lnurl-pay is not the same as Satoshi defined in another library

Reply:
you dont need to import the type, just use utils.checkedToSats or utils.toSats, also removed the url warning and included lnurlp in the new version 2.3.0 https://www.npmjs.com/package/lnurl-pay